next.config.mjs

```diff
@@ -8,6 +8,7 @@ const nextConfig = {
       },
     ],
   },
+  reactStrictMode: true,
```

Comment:
This is the default behavior for app router. Infact, you can check by noticing that all `useEffects` run twice.